coss/home/templates/home/about_page.html

```diff
@@ -15,46 +15,80 @@
   <div class="content">
 
     <nav id="nav-about" class="nav nav-about justify-content-center">
-      {% if page.about.body_text %}
+      {% if page.about.body_title %}
         <a class="nav-link" href="#body">{{ page.about.body_title }}</a>
       {% endif %}
-      {% if page.about.info_text %}
+      {% if page.about.info_title %}
         <a class="nav-link" href="#info">{{ page.about.info_title }}</a>
       {% endif %}
-      {% block extra_about_nav %}
-      {% endblock %}
+      {% if page.about.details_title %}
+        <a class="nav-link" href="#details">{{ page.about.details_title }}</a>
+      {% endif %}
     </nav>
 
     {% if page.about.body_text %}
-      <div id="body" class="body">
+      {% if page.about.body_image %}
+        {% image page.about.body_image original as bg_photo %}
+      {% endif %}
+      <div id="body" class="body" data-bg-img="{{ bg_photo.url }}">
```

Comment:
What happens if the `bg_photo` is undefined?

Reply:
In that case that data attribute is empty and the javascript code will fallback to the default one.